mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1135,7 +1137,7 @@ func resourceSqlDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
 	return nil
 }
 
-func expandSqlDatabaseInstanceSettings(configured []interface{}) *sqladmin.Settings {
+func expandSqlDatabaseInstanceSettings(configured []interface{}, databaseVersion string) *sqladmin.Settings {
```

Comment:
Maybe consider a comment? "Rules for settings can vary between database types" or similar.